process/rewardTransaction/process.go

```diff
@@ -90,9 +95,30 @@ func (rtp *rewardTxProcessor) ProcessRewardTransaction(rTx *rewardTx.RewardTx) e
 		return err
 	}
 
+	rtp.saveAccumulatedRewards(rTx, accHandler)
+
 	return rtp.accounts.SaveAccount(accHandler)
 }
 
+func (rtp *rewardTxProcessor) saveAccumulatedRewards(
+	rtx *rewardTx.RewardTx,
+	userAccount state.UserAccountHandler,
+) {
+	if !core.IsSmartContractAddress(rtx.RcvAddr) {
+		return
+	}
+
+	existingReward := big.NewInt(0)
+	fullRewardKey := core.ElrondProtectedKeyPrefix + rewardKey
```

Comment:
shouldn't expose this fullRewardKey through blockchain hook interface so a developer won't bother searching through our source code for constants?

Reply:
I think it would be enough to explain this in a tutorial and to have it in the docs.

Reply:
It is also easy to expose in the Rust API, no need for additional hook